README.md

```diff
@@ -1,100 +1,110 @@
 # A Minimal Meta-Transaction Library
```

Comment:
Some of the code examples in this readme are out of date

Reply:
Still some reference to old code

Reply:
Only problem I could see was one instance of the proxy contract having the wrong isContractDeployed() function. Let me know if there are any others.